### DISCUSSION TITLE
costs

### DISCUSSION THREAD

**Discussion by cleversmart** [DarwinsBuddy/WienerNetzeSmartmeter#290] [Q&A] [answered]
I am working successfully with Wienernergie integration 1.7.0 and would also like to have the costs displayed in the energy dashboard. What options are there for this? Do you have a Howto? Thanks!

**Comment by DarwinsBuddy** [comment, marked as answer]
usually you can easily define this within home assistant's energy dashboard configs. nothing that we should provide. 
[![Open your Home Assistant instance and show your Energy configuration panel.](https://my.home-assistant.io/badges/config_energy.svg)](https://my.home-assistant.io/redirect/config_energy/) 
When adding a consumption sensor you can select a way to track your costs.

or you could use a utility meter for a more sophisticated solution
https://www.home-assistant.io/integrations/utility_meter/
to track a tariff to be fed into the energy  dashboards configured sensors
